queries/nix/highlights.scm

```diff
@@ -0,0 +1,114 @@
+; basic keywords
+[
+  ("assert")
+  ("with")
+  ("let")
+  ("in")
+  ("rec")
+  ("inherit")
+] @keyword
+
+; if/then/else
+[
+  ("if")
+  ("then")
+  ("else")
+] @conditional
+
+; undocumented, I'm not actually sure what this does
+("or") @keyword.operator
```

Comment:
some wanted different highlights when operators were words instead of +,/,*,- . Default highlighting that "or" will have the same highlighting as `+`. `-`

Reply:
`@keyword.operator` is documented. The `or` operator is in Nix's lexer, and I recall having seen it used, but I can't find any documentation for what it actually does.

Reply:
...it's with all the other operators. I don't know how I missed it. It's used to specify a default for a field access. Normally, `a.b` will error if the field doesn't exist, but `a.b or c` will instead evaluate as `c`. I think `@keyword.operator` is fitting for that.